core/state/statedb.go

```diff
@@ -39,7 +39,10 @@ import (
 	"github.com/ethereum/go-ethereum/trie"
 )
 
-const defaultNumOfSlots = 100
+const (
+	defaultNumOfSlots          = 100
+	snapshotStaleRetryInterval = time.Millisecond * 100
```

Comment:
Please define `snapshotStaleRetryInterval` in `state_object.go`, and `time.Millisecond * 100` seems too large, `time.Millisecond * 10` is good enough.

Reply:
> Please define `snapshotStaleRetryInterval` in `state_object.go`, and `time.Millisecond * 100` seems too large, `time.Millisecond * 10` is good enough.

fixed